docs/about/whats_new.rst

```diff
@@ -40,7 +37,7 @@ Massive cleaning effort across the codebase, spearheaded by @pjonsson, with a fo
 Other Changes
 -------------
 
-- Use odc-geo GridSpec if `tile_shape` in product storage information :pull:`1783`
+- Use odc-geo GridSpec if `tile_shape` is in product storage information :pull:`1783`
 - SQL: fix package names type :pull:`1784`
 - Add some type signatures :pull:`1785`, :pull:`1788`, :pull:`1796`
 - Use Ruff instead of flake8 :pull:`1789`
```

Comment:
I think the upgrading user will have an easier time to sift through the relevant changes if you add a "CI Fixes and Improvements" heading at the end and move all the PRs that have "Dependabot" or "CI:" in their title. And PR 1890/1911 are also CI fixes.